src/services/utilities.ts

```diff
@@ -402,8 +402,11 @@ namespace ts {
         return start < end;
     }
 
+    /**
+     * Assumes `candidate.start <= position` holds.

```

Comment:
We don't have any conditional compilation. That is, if we wrote `Debug.assert(...)`, the enclosed expression would be run every time.

Since the assumed value isn't checked for performance reasons, it would defeat the purpose to add an assert that always runs.